addon/background.js

```diff
@@ -22,22 +22,30 @@ const DEFAULT_DESKTOP_VERSION = 1;
 
 const MAX_RECENT_TABS = 5;
 const manifest = browser.runtime.getManifest();
+const isShield = manifest.applications.gecko.id.endsWith("shield.mozilla.org");
 let sidebarUrl;
 let sidebarWidth;
 let hasSeenPrivateWarning = false;
 
-const ga = new TestPilotGA({
-  an: "side-view",
-  aid: manifest.applications.gecko.id,
-  aiid: "testpilot",
-  av: manifest.version,
-  // cd19 could also be dev or stage:
-  cd19: buildSettings.NODE_ENV === "prod" ? "production" : "local",
-  ds: "addon",
-  tid: buildSettings.NODE_ENV === "prod" ? "UA-77033033-7" : "",
-});
+let ga;
+if (!isShield) {
+  ga = new TestPilotGA({
+    an: "side-view",
+    aid: manifest.applications.gecko.id,
+    aiid: "testpilot",
+    av: manifest.version,
+    // cd19 could also be dev or stage:
+    cd19: buildSettings.NODE_ENV === "prod" ? "production" : "local",
+    ds: "addon",
+    tid: buildSettings.NODE_ENV === "prod" ? "UA-77033033-7" : "",
+  });
+}
 
 async function sendEvent(args) {
+  if (isShield) {
+    console.info("Aborting event for Shield");
```

Comment:
This seems like it will be spammy, and it doesn't provide useful info. Should it just be removed?

Reply:
Sure, I just wanted to keep it from being entirely silent right now, as we have to reintroduce events for Shield (but this patch doesn't do that).

Reply:
👌

i wondered why all events were ignored -- it's just not done yet.